templates/generic/object_form.html

```diff
@@ -23,6 +24,11 @@ <h1 class="pg-title">{{ title }}</h1>
         {% endblock form %}
         {% block form_actions %}
           <input type="submit" class="pg-button-primary mt-2" value="{{ button_text }}">
+          {% flag "experiment_versions" %}
+            {% if show_version_action_options %}
+              {% include 'experiments/experiment_version_edit_actions.html' %}
+            {% endif %}
+          {% endflag %}
```

Comment:
These changes should go in `experiment_form.html`. You can override the `form_actions` block.

That would also remove the need for the `show_version_action_options` var. You could instead check `object.id` to determine if the experiment is saved or not and use that as a display condition for the delete button

Reply:
Also, we probably want to show the delete button regardless of the `experiment_versions` flag status. Looks like it will only show when the flag is active